homeassistant/components/jellyfin/__init__.py

```diff
@@ -15,7 +15,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Jellyfin from a config entry."""
     hass.data.setdefault(DOMAIN, {})
 
-    client = create_client()
+    if entry.unique_id is None:
+        _LOGGER.error("Config entry was not created correctly")
```

Comment:
Can this actually happen, and if it can, what should the user do?
If it can happen, i.e. there was no unique_id for very old installations, you could extend the print to tell the user they need to remove the existing config entry and create a new one + add a comment explaining in which version of home assistant the unique_id was added.
If it can't happen because the entry has been given a unique_id from when jellyfin was first added, and the check is only there to satisfy mypy, please replace it with an assert.

Reply:
It was the latter indeed. I've replaced this with an assert.